pkgs/applications/version-management/gitlab/data.json

```diff
@@ -1,14 +1,22 @@
 {
-  "version": "15.1.4",
-  "repo_hash": "sha256-Ca3rhUJV7OVr3Uv6YmQxHspgwCKp1CZ4phUL1reREy8=",
-  "yarn_hash": "19df16gk0vpvdi1idqaakaglf11cic93i5njw0x4m2cnsznhpvz4",
+  "version": "15.2.1",
+  "repo_hash": "sha256-RDHq1wQaXAcf20GxH0ezG9YnYBr7SJZ8Hy3rzhaSODQ=",
+  "yarn_hash": "154akdngdagwfn8s90aw6sajw058i507shv5wzdn8l0vy3badgkv",
   "owner": "gitlab-org",
   "repo": "gitlab",
-  "rev": "v15.1.4-ee",
+  "rev": "v15.2.1-ee",
   "passthru": {
-    "GITALY_SERVER_VERSION": "15.1.4",
-    "GITLAB_PAGES_VERSION": "1.59.1",
-    "GITLAB_SHELL_VERSION": "14.7.4",
-    "GITLAB_WORKHORSE_VERSION": "15.1.4"
-  }
+    "GITALY_SERVER_VERSION": "15.2.1",
+    "GITLAB_PAGES_VERSION": "1.61.1",
+    "GITLAB_SHELL_VERSION": "14.9.0",
+    "GITLAB_WORKHORSE_VERSION": "15.2.1"
```

Comment:
Were these passthru version values ever used for anything? I went back to the original PR that introduced them and I don't see them in use even then. I think we can remove them?

Reply:
@yayayayaka I'd appreciate thoughts on this, if you have any :)